spec/models/comment_spec.rb

```diff
@@ -0,0 +1,6 @@
+require "rails_helper"
+
+describe Comment, :type => :model do
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)